examples/align-gl/rs-align-gl.cpp

```diff
@@ -124,11 +130,17 @@ int main(int argc, char * argv[]) try
         glDisable(GL_BLEND);
 
         // Render the UI:
-        ImGui_ImplGlfw_NewFrame(1);
+        ImGui_ImplOpenGL3_NewFrame();
```

Comment:
This can abe placed at realsense_imgui.h right as a 
push_new_frame
pop_new_frame

or something like this, helper funcs that will be used everywhere.

Thoughts?

Reply:
good idea I added PushNewFrame() and PushNewFrame() helper functions